doc/opam-provided.mld

```diff
@@ -25,26 +25,26 @@ should be just installed via [opam]. Such examples include:
 These users can opt-in to having some of their dependencies provided by [opam]
 instead of [opam-monorepo] pulling them into the [duniverse].
 
-{2 Initial setup}
+{2 Initial Setup}
 
 With the default configuration, [opam-monorepo] runs in full-[duniverse] mode,
-i.e., requiring all the dependencies of your [opam] files to be able to be
+i.e., requiring all the dependencies of your opam files to be able to be
 built as part of the [duniverse].
 
-As such it is only recommended that packages are installed via [opam] for which
+As such it's only recommended that packages are installed via [opam] for which
 there is no way to build them with [dune], e.g., by using the
 {{:https://github.com/dune-universe/opam-overlays}opam-overlays} repository
 with Dune ports.
 
 To let [opam-monorepo] know a package is to be installed via [opam], it needs
```

Comment:
"to be installed via `opam`" ... the command? or the package manager?
I suspect the command, so it's formatted correctly if that's right.